src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.20
+
+21 May 2024
+
+This version is compatible with Kubernetes versions 1.26 to 1.29
+
+### New Features
+
+- Add support for Openshift: Openshift Routes now show as Okteto Endpoints  <!-- 8025 -->
+- Enable Pull Secrets by default  <!-- 8001 -->
+- Added onboarding guides to our documentation on [how to get started with Okteto Insights](admin/okteto-insights.mdx) <!-- 729 -->
+
+### Improvements
+
+- Namespaces without an owner will now be picked up by the Garbage Collector  <!-- 8018 -->
+- Added the ability to assign a PriorityClass to Okteto components  <!-- 7966 -->
```

Comment:
Should we indicate how his can be done? (through helm settings)

Reply:
Should we link to the property in the helm chart `globals.priorityClassName`, then? Should we also mention how do they work (preference) with the individual priorityClass you can define for Okteto components?

Reply:
What section would we link to here?

Reply:
This would be the link: https://www.okteto.com/docs/1.20/self-hosted/helm-configuration/#globals